src/main/java/teammates/common/util/Const.java

```diff
@@ -876,6 +876,9 @@ private Const() {
         public static final String STUDENT_FEEDBACK_RESULTS_PAGE = "/page/studentFeedbackResultsPage";
         public static final String INSTRUCTOR_FEEDBACK_SUBMISSION_EDIT_PAGE = "/page/instructorFeedbackSubmissionEditPage";
         public static final String INSTRUCTOR_FEEDBACK_RESULTS_PAGE = "/page/instructorFeedbackResultsPage";
+        public static final String INSTRUCTOR_EDIT_STUDENT_FEEDBACK_PAGE = "/page/instructorEditStudentFeedbackPage";
```

Comment:
We are in migration process so these `*PAGE` references are supposed to be deleted. You should not added it back. For places where it reference such variable. You should leave them there.

Reply:
so that means we need to leave the error as it is ???

Reply:
Yes. As the file will be gradually migrated. There is no point to update them.